esmvaltool/cmorizers/obs/download_scripts/download_cds_satellite_soil_moisture_monthly.py

```diff
@@ -0,0 +1,159 @@
+"""
+Download cds-satellite-soil-moisture.
+
+Script to download cds-satellite-soil-moisture
+from the Climate Data Store(CDS).
+"""
+
+import argparse
+import os
+import subprocess
+
+import yaml
+
+import cdsapi
+
+
+def download_cds_satellite_soil_moisture():
+    """Download dataset cds-satellite-albedo."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--config_file',
+                        '-c',
+                        default=os.path.join(os.path.dirname(__file__),
+                                             'config-user.yml'),
+                        help='Config file')
+    parser.add_argument('--frequency',
+                        '-f',
+                        default=None,
+                        help="Either monthly or daily")
+
+    args = parser.parse_args()
+
+    # get and read config file
+    config_file_name = os.path.abspath(
+        os.path.expandvars(os.path.expanduser(args.config_file)))
+
+    with open(config_file_name, 'r') as config_file:
+        config = yaml.safe_load(config_file)
+
+    rawobs_dir = os.path.abspath(
+        os.path.expandvars(os.path.expanduser(config['rootpath']['RAWOBS'])))
+    savedir = f'{rawobs_dir}/Tier3/CDS-SATELLITE-SOIL-MOISTURE_tmp/'
```

Comment:
```suggestion
    savedir = f'{rawobs_dir}/Tier3/CDS-SATELLITE-SOIL-MOISTURE/'
```
Let's save them directly in the right directory.